tests/testthat/test-linear_pool.R

```diff
@@ -679,3 +679,20 @@ test_that("ensemble of samples throws an error for the more complex cases", {
     "`weights` must be \"NULL\" or equal for every model", fixed = TRUE
   )
 })
+
+test_that("Arg `derived_tasks` is deprecated", {
+  sample_outputs <- create_test_sample_outputs() |>
+    dplyr::mutate(reference_date = target_date - 7 * horizon)
+  expect_snapshot({
```

Comment:
To suppress warnings, you can use `suppressWarnings` instead of wrapping it in a snapshot. 

```suggestion
  suppressWarnings({
```

Reply:
Out of curiosity, why is suppressing warnings considered better in this case?

Reply:
Because the purpose of the snapshot is to test for output that's easier to visually inspect than to test programmatically. 

In practice, I've seen so many people use these tests as a shortcut to testing validity and it leads to situations like https://github.com/hubverse-org/hubValidations/issues/168.

Thinking about this, from what [the documentation says](https://testthat.r-lib.org/reference/expect_error.html), you shouldn't even need the `suppressWarnings()` since the test should capture it.

Reply:
Okay, thank you!